docs/changelog.md

```diff
@@ -4,7 +4,7 @@
 
 # Changelog
 
-## Unreleased
+## 0.67.0 (April 2025)
 
 - Add support for Python 3.13.
 - Support qiskit >=2.0, drop support for qiskit <2.0.
```

Comment:
Maybe we should say explicitly that pytket condtional gates are now handled as `IfElseOp` instances in qiskit?

Reply:
I think that's a detail that doesn't need to go in the changelog.